### ISSUE TITLE
sarco lids and floor candelabra should not be inf intertia pushable

### ISSUE THREAD

**Issue by Mooses2k** [Mooses2k/SecretHistories#401]
Something about making the lid and floor candelabras a KinematicBody when its sleeping or something?
* player needs it to be a RigidBodywhen grabbed
* It still needs to obey physics when its tipped over the edge, it needs to fall, but it should NOT be pushable with infinite intertia by player's KinematicBody

~~Either slow player based on weight of object or how close they are to end of drag distance to simulate heaviness of moving something. Maybe use the amount of force grab is putting on the object?~~

~~Sarco lids, for example, can be dragged in dev but its a mess cuz the player keeps moving too fast out of the range.~~

**Comment by Mooses2k** [comment]
Done well in branch. Main issue remaining: since we switched away from a RigidBody player (during stealth player controller import), we've got the infinite inertia issue pushing stuff around. With doors, we prefer player pushing them, with sarco lids, the opposite. Any way to set those so they act as static when pushed by characters? They need to be able to move to fall to the ground when let go of grab, so it can't just be 'be active while grabbed'. Thoughts? This will be a demo blocker.
